packages/odyssey-react/src/components/Tab/Tab.module.scss

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.tablist {
+  position: relative;
+  border-block-end: 1px solid $border-color-display;
+}
+
+.tab {
```

Comment:
maybe, `.root`, `.list`, `.panel` or `.tab`, `.list`, `.panel`?

Reply:
I buy that approach here. Despite the name, "Tab" is not nec a root component. Maybe we can come up with clear guidelines on when to use which convention.